d2bs/kolbot/libs/bots/MFHelper.js

```diff
@@ -228,13 +229,17 @@ MainLoop:
 						delay(1000);
 					}
 
-					if (me.area === area) {
+					if (me.area === player.area) {
 						Precast.doPrecast(false);
 						Attack.clearLevel(Config.ClearType);
 						Precast.doPrecast(true);
 
 						if (!Pather.usePortal(null, player.name)) {
 							Town.goToTown();
+							if (Config.MFHelper.HealBetweenCommands) {
```

Comment:
Town.heal checks config options via "needHealing" therefore, separate config not needed for healbetweencommands